### PR TITLE
Assign task to a task list when creating task

### LINKED ISSUE #31: Assign task to task list
Allows users to assign tasks to tasks lists

### PR DESCRIPTION
**Pull request by AaronDzf**
• Added a function to assign a task to a task list when creating a task using a drop-down menu
• SQL query connects to the database to retrieve a list of all task lists for the drop-down menu
• Updated the task insert query to include user chosen task list
• Removed "BoardView.html" as it is no longer in use - replaced by "listView.php"
• Removed "taskDetails.html" as it is no longer in use - replaced by "taskDetails.php"
• Fixed typos to variable names

Testing:
(Ensure connection to the mansci database is stable, via VPN if needed)
Open the taskDetails.php page on browser. 
Compare and check the list of task lists in the drop-down menu has all the task lists that exist in the database. 
Add details of a temporary task and select any task list. Check the database and see if the newly added task is in the correctly selected task list. If unable to access the database, return to listView.php and verify that the newly added task exists in the correct task list (Assume that listView.php has no bugs in displaying the tasks).

### CODE REVIEW

File: taskDetails.php [hunk 1]
```diff
@@ -1,31 +1,91 @@
...
+$submitted_taskListAssigned = $_POST['listID'];
 
 //imports the objects and valiues from conn
 require("conn.php"); 
-$sql = "INSERT INTO `task` (`taskID`, `taskTitle`, `description`, `dueDate`, `taskDateCreated`, `importance`, `typeOfWork`, `c.courseID`, `tl.listID`, `archived`) VALUES (null, '$submitted_tastName', '$submitted_taskDetail', $submitted_taskDateDue , '10', $submitted_taskEffort, '$submitted_taskTypeWork', '111', '1', '1');";
+$sql = "INSERT INTO `task` (`taskID`, `taskTitle`, `description`, `dueDate`, `taskDateCreated`, `importance`, `typeOfWork`, `c.courseID`, `tl.listID`, `archived`) 
+        VALUES (null, '$submitted_tastName', '$submitted_taskDetail', $submitted_taskDateDue , '10', $submitted_taskEffort, '$submitted_taskTypeWork', '111', $submitted_taskListAssigned , '1');";
 //executes the post call
 $result = mysqli_query($conn, $sql);
-echo $result;
```
Comment (Rolo123y) [inline taskDetails.php]: should be in PDO format rather than mysqli

Comment (Rolo123y) [reply taskDetails.php]: Can be changed later if not possible now.

File: taskDetails.php [hunk 2]
```diff
@@ -1,31 +1,91 @@
...
-echo $submitted_taskEffort;
-echo $submitted_taskDateDue;
-echo $submitted_taskTypeWork;
+$submitted_taskListAssigned = $_POST['listID'];
 
 //imports the objects and valiues from conn
 require("conn.php"); 
-$sql = "INSERT INTO `task` (`taskID`, `taskTitle`, `description`, `dueDate`, `taskDateCreated`, `importance`, `typeOfWork`, `c.courseID`, `tl.listID`, `archived`) VALUES (null, '$submitted_tastName', '$submitted_taskDetail', $submitted_taskDateDue , '10', $submitted_taskEffort, '$submitted_taskTypeWork', '111', '1', '1');";
+$sql = "INSERT INTO `task` (`taskID`, `taskTitle`, `description`, `dueDate`, `taskDateCreated`, `importance`, `typeOfWork`, `c.courseID`, `tl.listID`, `archived`) 
+        VALUES (null, '$submitted_tastName', '$submitted_taskDetail', $submitted_taskDateDue , '10', $submitted_taskEffort, '$submitted_taskTypeWork', '111', $submitted_taskListAssigned , '1');";
```
Comment (Rolo123y) [inline taskDetails.php]: $submitted_tastName should be changed to $submitted_taskName

**Review by Rolo123y — CHANGES_REQUESTED** [review]
removing boardView and taskDetails.html is good. $submitted_tastName should be changed to $submitted_taskName on line 86 of taskDetails.php

**Review by Rolo123y — APPROVED** [review]

File: taskDetails.php (hunk 2 above)
Comment (AaronDzf) [reply taskDetails.php]: Thanks for the catch! will update asap.

**Pull request merged**